docs/conf.py

```diff
@@ -64,8 +64,10 @@
 autodoc_class_signature = "separated"
 autodoc_default_options = {
     "members": True,
+    "inherited-members": True,
     "member-order": "bysource",
```

Comment:
On the existing configuration, there are a few options. I believe the methods can be ordered by `source` or `alphabetical`. Unfortunately, we don't have more control about this in my understanding than this parameter. I had intended to give it some time iterating a few approaches, but then realised we might already be limited on this. 

I've raised an issue for the new docs extension to enable this https://github.com/flexcompute/autoflex/issues/12

However, maybe for now this is it?

Reply:
one or the other is probably fine enough for now.